startmicro

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+regex="([^\/]+)-([^-]+)-cache-config.yaml"
+node_yaml='(() => {
+var yaml = require("js-yaml");
+var fs   = require("fs");
+var doc  = yaml.safeLoad(fs.readFileSync(process.argv[1], "utf8"));
```

Comment:
Sorry, I meant that `key` and `secret` are passed as commandline arguments.  This change would effectively require they be provided in the cached file, right?  If I remember correctly, the key and secret are by default written to the cached file.  If they aren't, then this would be breaking, right?